### PR TITLE
tetragon: docs, trnasform events into single page

### PR DESCRIPTION
**Pull request by jrfastab**
I don't like these multi page click icons lets just build a single page for events.

### CODE REVIEW

File: docs/content/en/docs/concepts/events.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+Tetragon's events are exposed to the system through either the gRPC
+endpoint or JSON logs. Commands in this section assume the Getting
+Started guide was used, but are general other than the namespaces
+chosen and should work in most environments.
+
+### JSON
+
+The first way is to observe the raw json output from the stdout container log:
+
+```shell
```
Comment (mtardy) [inline docs/content/en/docs/concepts/events.md:17]:
```suggestion
```shell-session
```

File: docs/content/en/docs/concepts/events.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+GOARCH=$(go env GOARCH)
+curl -L --remote-name-all https://github.com/cilium/tetragon/releases/latest/download/tetra-${GOOS}-${GOARCH}.tar.gz{,.sha256sum}
+sha256sum --check tetra-${GOOS}-${GOARCH}.tar.gz.sha256sum
+sudo tar -C /usr/local/bin -xzvf tetra-${GOOS}-${GOARCH}.tar.gz
+rm tetra-${GOOS}-${GOARCH}.tar.gz{,.sha256sum}
+```
+
+See the [latest release
+page](https://github.com/cilium/tetragon/releases/latest) for supported
+`GOOS`/`GOARCH` binary releases.
```
Comment (mtardy) [inline docs/content/en/docs/concepts/events.md:147]: I think we should remove that part and link to the install guide for tetra CLI directly. It's duplicated info

File: docs/content/en/docs/concepts/events.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+To start printing events run:
+```shell
```
Comment (mtardy) [inline docs/content/en/docs/concepts/events.md:150]:
```suggestion
```shell-session
```

File: docs/content/en/docs/concepts/events.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+kubectl logs -n kube-system -l app.kubernetes.io/name=tetragon -c export-stdout -f | tetra getevents -o compact
+```
+
+The `tetra` CLI is also available inside `tetragon` container.
+
+```shell
```
Comment (mtardy) [inline docs/content/en/docs/concepts/events.md:156]:
```suggestion
```shell-session
```

File: docs/content/en/docs/concepts/events.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+kubectl exec -it -n kube-system ds/tetragon -c tetragon -- tetra getevents -o compact
+```
+
+This was used in the quick start and generates a pretty printing of the events, To further
+filter by a specific binary and/or pod do the following,
+
+```shell
```
Comment (mtardy) [inline docs/content/en/docs/concepts/events.md:163]:
```suggestion
```shell-session
```

File: docs/content/en/docs/concepts/events.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+### gRPC
+
+In addition Tetragon can expose a gRPC endpoint listeners may attach to. The
+gRPC is exposed by default helm install on `localhost:54321`, but the address
+can be configured  with the `--server-address` option. This can be
+set from helm with the `tetragon.grpc.address` flag or disabled completely if
+needed with `tetragon.grpc.enabled`.
+
+```shell
```
Comment (mtardy) [inline docs/content/en/docs/concepts/events.md:182]:
```suggestion
```shell-session
```

File: docs/content/en/docs/concepts/events.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+helm install tetragon cilium/tetragon -n kube-system --set tetragon.grpc.enabled=true --set tetragon.grpc.address=localhost:54321
+```
+
+An example gRPC endpoint is the Tetra CLI when its not piped JSON output directly,
+
+```shell
```
Comment (mtardy) [inline docs/content/en/docs/concepts/events.md:188]:
```suggestion
```shell-session
```

**Review by mtardy — APPROVED** [review]
Yes I completely agree, I didn't understand where this split came from originally :)!

If you have time, I have some comments however.

**Comment by jrfastab** [comment]
thanks Mahe I think I resolved your comments.

**Pull request merged**